pkg/utils/k8sutil.go

```diff
@@ -49,7 +49,7 @@ const (
 
 // GetClient returns a k8s clientset to the request from inside of cluster
 func GetClient() kubernetes.Interface {
-	config, err := utils.GetInClusterConfig()
+	config, err := rest.InClusterConfig()
```

Comment:
why have you changed this line?

Reply:
tbh, my first try at go et-all (beside some hello world stuff) - it complained that `utils.GetInClusterConfig()`  is undefined, and found a similar usage on kubeless/kubless and used the `rest.` instead, which seem'd to work.

i am open for suggestions.

Reply:
`utils.GetInClusterConfig` is a wrapper around the `rest.InClusterConfig` that allows to override the token used based in an env variable. You should be able to use it (at least now, I don't know about the error you got), mostly because it's being called a bit below (line 67)